src/compiler/scanner.ts

```diff
@@ -2242,7 +2242,7 @@ namespace ts {
             return token = SyntaxKind.QuestionToken;
         }
 
-        function scanJsxToken(): JsxTokenSyntaxKind {
+        function scanJsxToken(isFormatting?: boolean): JsxTokenSyntaxKind {

```

Comment:
I would suggest renaming and flipping this boolean (and the one in `reScan`) so it’s explicit about what it’s doing:

```suggestion
        function scanJsxToken(allowMultilineJsxText = true): JsxTokenSyntaxKind {
```

Reply:
Renamed.